src/main/java/com/orangehrm/pages/MyInfo.java

```diff
@@ -0,0 +1,35 @@
+package com.orangehrm.pages;
+
+import com.codeborne.selenide.SelenideElement;
+import static com.codeborne.selenide.Condition.*;
+import com.codeborne.selenide.SelenideElement;
+import com.orangehrm.entity.InvalidLoginDetails;
+import com.orangehrm.entity.ValidLoginDetails;
+
+import static com.codeborne.selenide.Selectors.*;
+import static com.codeborne.selenide.Selenide.$;
+
+public class MyInfo {
+    private static final SelenideElement FIRST_NAME= $(byName ( "firstName" ));
+    private static final SelenideElement MIDDLE_NAME= $(byName ( "middleName" ));
+    private static final SelenideElement LAST_NAME= $(byName ( "lastName" ));
+    private static final SelenideElement EMPLOYEE_ID= $(byXpath ( "(//input[@class='oxd-input oxd-input--active'])[3]" ));
+    private static final SelenideElement OTHER_ID= $(byXpath ( "(//input[@class='oxd-input oxd-input--active'])[4]" ));
+    private static final SelenideElement DRIVER_LICENSE_NUMBER= $(byXpath ( "(//input[@class='oxd-input oxd-input--active'])[5]" ));
+    private static final SelenideElement LICENSE_EXPIRY_DATE =$(byAttribute ( "placeholder","yyyy-mm-dd" ));
+    private static final SelenideElement SAVE_BUTTON = $(byText ( "Save" ));
+
+
+    public MyInfo myDetails(){
```

Comment:
All the hardcoded data has been added, please update with generic one.
you can use some libraries to fix this